esignet-integration-api/src/main/java/io/mosip/esignet/api/spi/VCIssuancePlugin.java

```diff
@@ -0,0 +1,22 @@
+package io.mosip.esignet.api.spi;
+
+import io.mosip.esignet.api.dto.VCRequestDto;
+import io.mosip.esignet.api.dto.VCResult;
+
+import java.util.Map;
+
+/**
+ *
+ */
+public interface VCIssuancePlugin {
+
+    /**
+     * WIP
+     * @param vcRequestDto
+     * @param holderKey
+     * @param identityDetails parsed access-token or introspect endpoint response if token is opaque.
+     * @return
+     */
+    VCResult getVerifiableCredential(VCRequestDto vcRequestDto, String holderKey,
```

Comment:
holderKey - can we not call this as holderId ? Because this might not be a Key in reality

Reply:
Done